.github/workflows/scripts/check_ignore.py

```
@@ -0,0 +1,259 @@
+        "--directory",
+        type=str,
+        required=False,
+        default=os.getcwd(),
+        help="The parent directory of files to analyze.",
+    )
+
+    # Return parser
+    return parser.parse_args()
+
+
+def _filepaths_in_directories(directories):
```

Ruff (0.11.9)

Missing return type annotation for private function `_filepaths_in_directories`

(ANN202)